feat(guide): Update naming scheme and add tvdb/tmdb (#1965)
TRaSH-Guides/Guides: docs/Radarr/Radarr-recommended-naming-scheme.md

## Standard Movie Format

> IMDb IDs are very accurate and rarely change. In contrast, TMDb IDs do change or are removed more frequently, but they are more aligned with Radarr.

Reviewer: ```suggestion
While both IMDb and TMDb IDs are unique, TMDb can occasionally remove IDs entirely, sometimes only to be readded with a new ID later. However, due to using TMDb as its metadata source, they can be seen as "more aligned" with Radarr. IMDb IDs on the other hand, once present, are very accurate and rarely ever change. 
```
Not super sure on this, but this seems good to me like this, thoughts? @yammes08 ?

=== "Standard"

>     ```bash

Reviewer 2: ```suggestion
    ```none
```

    This naming scheme is made to be used with the [New Plex Agent](https://forums.plex.tv/t/new-plex-media-server-movie-scanner-and-agent-preview/593269/517){:target="\_blank" rel="noopener noreferrer"},

    !!! danger "If you use the `{edition-{Edition Tags}}` part of the recommended file name, then Plex will recognize the edition of the movie and add it the the Plex UI - e.g., Directors Cut.<br><br>However, this means that if you have two copies of a movie with different editions in a single merged library - for example, a 1080p Directors Cut and a 2160p Theatrical Edition - then these will show as two separate items in Plex.<br><br>If you only want a movie to appear once per library, where you keep more than one copy of a movie, then replace: `{edition-{Edition Tags}}` with `{Edition Tags}`."

    === "Plex (IMDb)"

        --8<-- "includes/radarr/tmdb-imdb-info.md"

        ```bash
        {{ radarr['naming']['radarr-naming']['file']['plex-imdb'] }}
        ```

        <sub><sup>**Movie**: `The Movie Title (2010) {imdb-tt0066921} {edition-Ultimate Extended Edition} [IMAX HYBRID][Bluray-1080p Proper][3D][DV HDR10][DTS 5.1][x264]-EVOLVE`</sup></sub>

    === "Plex (TMDb)"

>         ```bash

Reviewer 2: ```suggestion
        ```none
```

        {{ radarr['naming']['radarr-naming']['file']['plex-tmdb'] }}
        ```

        <sub><sup>**Movie**: `The Movie Title (2010) {tmdb-345691} {edition-Ultimate Extended Edition} [IMAX HYBRID][Bluray-1080p Proper][3D][DV HDR10][DTS 5.1][x264]-EVOLVE`</sup></sub>

    === "Plex Anime (IMDb)"

        --8<-- "includes/radarr/tmdb-imdb-info.md"

        ```bash
        {{ radarr['naming']['radarr-naming']['file']['plex-anime-imdb'] }}
        ```

        <sub><sup>**Movie**: `The Movie Title (2010) {imdb-tt0066921} {edition-Ultimate Extended Edition} [Surround Sound x264][Bluray-1080p Proper][3D][DTS 5.1][DE][10bit][AVC]-EVOLVE`</sup></sub>

    === "Plex Anime (TMDb)"

        ```bash
        {{ radarr['naming']['radarr-naming']['file']['plex-anime-tmdb'] }}
        ```

        <sub><sup>**Movie**: `The Movie Title (2010) {tmdb-345691} {edition-Ultimate Extended Edition} [Surround Sound x264][Bluray-1080p Proper][3D][DTS 5.1][DE][10bit][AVC]-EVOLVE`</sup></sub>

=== "Emby"

>     Used Source: [Emby Wiki/Docs](https://emby.media/support/articles/Movie-Naming.html#id-tags-in-folder--file-names){:target="\_blank" rel="noopener noreferrer"}

Reviewer: ```suggestion
    Source: [Emby Wiki/Docs](https://emby.media/support/articles/Movie-Naming.html#id-tags-in-folder--file-names){:target="\_blank" rel="noopener noreferrer"}

    === "Emby (TMDb)"

        ```bash
        {{ radarr['naming']['radarr-naming']['file']['emby-tmdb'] }}
        ```

        <sub><sup>**Movie**: `The Movie Title (2010) [tmdbid-tt0066921] - {edition-Ultimate Extended Edition} [IMAX HYBRID][Bluray-1080p Proper][3D][DV HDR10][DTS 5.1][x264]-EVOLVE`</sup></sub>

    === "Emby Anime (IMDb)"

        --8<-- "includes/radarr/tmdb-imdb-info.md"

        ```bash
        {{ radarr['naming']['radarr-naming']['file']['emby-anime-imdb'] }}
        ```

        <sub><sup>**Movie**: `The Movie Title (2010) [imdbid-tt0066921] - {edition-Ultimate Extended Edition} [Surround Sound x264][Bluray-1080p Proper][3D][DTS 5.1][DE][10bit][AVC]-EVOLVE`</sup></sub>

    === "Emby Anime (TMDb)"

        ```bash
        {{ radarr['naming']['radarr-naming']['file']['emby-anime-tmdb'] }}
        ```

        <sub><sup>**Movie**: `The Movie Title (2010) [tmdbid-tt0066921] - {edition-Ultimate Extended Edition} [Surround Sound x264][Bluray-1080p Proper][3D][DTS 5.1][DE][10bit][AVC]-EVOLVE`</sup></sub>

=== "Jellyfin"

>     Used Source: [Jellyfin Wiki/Docs](https://jellyfin.org/docs/general/server/media/movies){:target="\_blank" rel="noopener noreferrer"}

Reviewer: ```suggestion
    Source: [Jellyfin Wiki/Docs](https://jellyfin.org/docs/general/server/media/movies){:target="\_blank" rel="noopener noreferrer"}

## Movie Folder Format

> !!! warning "Please note that folder names are created in the database whenever the movie is added to Radarr, and it may be missing or incorrect at that time, resulting in your folder having a blank ID.:bangbang:<br><br>If you instead add it to the filename, the IMDb/TMDb ID will be freshly pulled for any download or upgrade."

Reviewer: ```suggestion
!!! warning "Please note that folder names are created in the database whenever the movie is added to Radarr and it may be missing or incorrect at that time, resulting in your folder having a blank ID.:bangbang:<br><br>If you instead add it to the filename, the IMDb/TMDb ID will be freshly pulled for any download or upgrade."

=== "Standard Folder"

>     Minimal needed and recommended

Reviewer: ```suggestion
    Minimum needed and recommended
